Nos/Service/Relay/RelayService.swift

```diff
@@ -19,11 +19,10 @@ class RelayService: ObservableObject {
     private var processingQueue = DispatchQueue(label: "RelayService-processing", qos: .userInitiated)
     private var parseQueue = ParseQueue()
     
-    @Dependency(\.persistenceController) var persistenceController
-    @Dependency(\.analytics) private var analytics
-    @Dependency(\.crashReporting) private var crashReporting
-    @MainActor @Dependency(\.currentUser) private var currentUser
-    @Published var numberOfConnectedRelays: Int = 0
```

Comment:
This property wasn't being used anywhere.

Reply:
Oh yeah. We used to have UI to display it but we removed it.